packages/plugins/Web3Profile/src/SNSAdaptor/api.ts

```diff
@@ -0,0 +1,25 @@
+import urlcat from 'urlcat'
+import { Response, AssetType } from './types'
+
+async function fetchJSON<T = unknown>(url: string): Promise<T> {
```

Comment:
```suggestion
async function fetchJSON(url: string): Promise<unknown> {
```

Don't cast, but check it's actual structure.